src/installer/installer_test.go

```diff
@@ -1,16 +1,40 @@
 package installer
 
-//import (
-//	"github.com/eranco74/assisted-installer/src/config"
-//	"github.com/sirupsen/logrus"
-//)
-//
-//var _ = Describe("installer", func() {
-//	var conf = config.Config{Role: "master", ClusterID: "cluster-id", Device: "/dev/vda", S3EndpointURL: "https://s3.com", S3Bucket: "test"}
-//	var l = logrus.New()
-//	i := NewAssistedInstaller(l, conf)
-//
-//	It("master role happy flow", func() {
-//		i.InstallNode()
-//	})
-//})
+import (
+	"github.com/eranco74/assisted-installer/src/config"
+	"github.com/eranco74/assisted-installer/src/ops"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"testing"
+)
+
+func TestValidator(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "inventory_test")
+}
+
+var _ = Describe("installer", func() {
+	var conf = config.Config{Role: "master", ClusterID: "cluster-id", Device: "/dev/vda", S3EndpointURL: "https://s3.com", S3Bucket: "test"}
+	var l = logrus.New()
+	l.SetOutput(ioutil.Discard)
+	ctrl := gomock.NewController(GinkgoT())
```

Comment:
usually i'm creating the controller before each test and need `ctrl.Finish()` otherwise you will not enforce anything.
and if you configure it once then you will validate all the mocks together and not for each test. if you have a bug it will be confusing to look at.